src/components/Header/Header.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+import {useNavigate} from 'react-router-dom';
+import { 
+    HeaderContainer,
+    NavigationHeader,
+    HeaderBackButton,
+    HeaderTitle,
+} from './Header.styled';
+
+interface HeaderProps {
+    title: string;
+    showBackButton?: boolean;
+}
+
+const Header = (props: HeaderProps): JSX.Element => {
+    const {title, showBackButton = true} = props;
```

Comment:
destructuring the `props` in the component, instead of in the functional component arguments